components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementFacade.java

```diff
@@ -323,6 +323,12 @@ public List<FederatedAuthenticatorConfig> getAllUserDefinedFederatedAuthenticato
         return configList;
     }
 
+    public void deleteIdpProperties(String tenantDomain, int idpId, List<String> propertyNames)
```

Comment:
better to move the tenantDomain to 3rd argument.

idpId, propertyNames, tenantDomain